.github/workflows/romcal-ci.yml

```
@@ -30,10 +30,16 @@ jobs:
         if: github.event_name == 'push'
         run: npx commitlint --last --verbose
 
-      - name: Validate PR commits with commitlint
+      - name: Validate with commitlint (pull_request)
         if: github.event_name == 'pull_request'
         run: npx commitlint --from ${{ github.event.pull_request.head.sha }}~${{ github.event.pull_request.commits }} --to ${{ github.event.pull_request.head.sha }} --verbose
 
+      - name: Lint Title (pull_request)
```

:pick: How about this? We don’t use Title Case in the CI action names, therefore, I think we should stick with this convention (unless you re-case them all). And `title` is quite general term, therefore, I personally miss some level of specificity.

```suggestion
      - name: Lint PR titles (pull_request)
```

---

there aren't any other Titles and we already specify `pull_request` as the scope. this is a distinction without a difference. i'd rather just remove the parenthetical, but this helps it be clear on a non-PR push